src/main/kotlin/net/maizegenetics/phgv2/api/HaplotypeGraph.kt

```diff
@@ -114,6 +114,14 @@ class HaplotypeGraph(hvcfFiles: List<String>) {
         return result
     }
 
+    /**
+     * Simple function to make a map of all the haplotypeIds and all the SampleGametes which have that haplotypeId.
```

Comment:
Is this included as a  more concise version of the original hapIdsToSamleGametes() function that previously existing in HaplotypeGraph?

Reply:
Yes, it allows users to just get the whole map if need be without having to loop through the refRanges.